onchain/rollups/test/foundry/dapp/CartesiDApp.t.sol

```diff
@@ -28,16 +28,22 @@ contract EtherReceiver {
     receive() external payable {}
 }
 
-// Outputs
-// 0: notice 0xfafafafa
-// 1: voucher ERC-20 transfer
-// 2: voucher ETH withdrawal
-// 3: voucher ERC-721 transfer
-
 contract CartesiDAppTest is TestBase {
     using LibServerManager for LibServerManager.RawFinishEpochResponse;
     using LibServerManager for LibServerManager.Proof;
 
+    // Outputs
+    // 0: notice 0xfafafafa
+    // 1: voucher ERC-20 transfer
+    // 2: voucher ETH withdrawal
+    // 3: voucher ERC-721 transfer
+    enum OutputCase {
+        Notice,
+        ERC20Voucher,
+        ETHVoucher,
+        ERC721Voucher
```

Comment:
I would add the word `Transfer` after `ERC20` and `ERC721`, and the word `Withdrawal` after `ETH`.